docs-ref-conceptual/spring-framework/configure-spring-boot-java-applicationinsights.md

```diff
@@ -0,0 +1,314 @@
+---
+title: Configure a Spring Boot Initializer app to use Azure Application Insights SpringBoot Starter
+description: Configure a Spring Boot application created with the Spring Initializr to use Appliaction Insights SpringBoot Starter.
+services: Application-Insights
+documentationcenter: java
+author: dhaval24
+manager: alexklim
+editor: ''
+
+ms.assetid:
+ms.author: dhdoshi
+ms.date: 05/19/2018
+ms.devlang: java
+ms.service: Azure Monitor
+ms.tgt_pltfrm: application-insights
+ms.topic: article
+ms.workload: na
+---
+
+# Configure a Spring Boot Initializer app to use Application Insights with Azure Application Insights Spring Boot Starter (PUBLIC PREVIEW)
+
+This article walks you through creating a Spring Boot application using **[Spring Initializr]**, that uses Azure Application Insights Spring Boot Starter for end-to-end monitoring of Java applications on cloud. *Please note this starter is currently in **BETA (public preview)**.*
```

Comment:
Please use the NOTE syntax (see below in this article) to point out that this starter is in public preview.